flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserTypeConverter.java

```diff
@@ -289,6 +289,7 @@ private static TypeInfo convertPrimitiveType(RelDataType rType) {
             case INTERVAL_SECOND:
                 return hiveShim.getIntervalDayTimeTypeInfo();
             case BINARY:
+            case VARBINARY:
                 return TypeInfoFactory.binaryTypeInfo;
```

Comment:
I find that there exist other types which is not present in flink code, for example, REAL, TIME_WITH_LOCAL_TIME_ZONE. Should we also add them?

Reply:
Thanks for your reminer. 
Yes, we also should all them. And we have also a jira for TIME_WITH_LOCAL_TIME_ZONE [FLINK-23224](https://issues.apache.org/jira/browse/FLINK-23224).